--- MOSAIX_ori/RunoffWeights.py	2021-03-29 13:30:01.000000000 +0200
+++ MOSAIX_docker/RunoffWeights.py	2021-03-29 13:37:23.000000000 +0200
@@ -312,8 +312,8 @@
 v_atm_grid_center_lat = f_runoff.createVariable ( 'src_grid_center_lat', 'i4', ('src_grid_size',) )
 v_atm_grid_center_lon.units='degrees_east'  ; v_atm_grid_center_lon.long_name='Longitude' ; v_atm_grid_center_lon.long_name='longitude' ; v_atm_grid_center_lon.bounds="src_grid_corner_lon"
 v_atm_grid_center_lat.units='degrees_north' ; v_atm_grid_center_lat.long_name='Latitude'  ; v_atm_grid_center_lat.long_name='latitude ' ; v_atm_grid_center_lat.bounds="src_grid_corner_lat"
-v_atm_grid_corner_lon = f_runoff.createVariable ( 'src_grid_corner_lon', 'f8', ('src_grid_size', 'src_grid_corners')  )
-v_atm_grid_corner_lat = f_runoff.createVariable ( 'src_grid_corner_lat', 'f8', ('src_grid_size', 'src_grid_corners')  )
+v_atm_grid_corner_lon = f_runoff.createVariable ( 'src_grid_corner_lon', 'f8', ('src_grid_corners', 'src_grid_size')  )
+v_atm_grid_corner_lat = f_runoff.createVariable ( 'src_grid_corner_lat', 'f8', ('src_grid_corners', 'src_grid_size')  )
 v_atm_grid_corner_lon.units="degrees_east"
 v_atm_grid_corner_lat.units="degrees_north"
 v_atm_grid_area       = f_runoff.createVariable ( 'src_grid_area'      , 'f8', ('src_grid_size',)  )
@@ -339,8 +339,8 @@
 v_oce_grid_center_lat = f_runoff.createVariable ( 'dst_grid_center_lat', 'i4', ('dst_grid_size',) )
 v_oce_grid_center_lon.units='degrees_east'  ; v_oce_grid_center_lon.long_name='Longitude' ; v_oce_grid_center_lon.long_name='longitude' ; v_oce_grid_center_lon.bounds="dst_grid_corner_lon"
 v_oce_grid_center_lat.units='degrees_north' ; v_oce_grid_center_lat.long_name='Latitude'  ; v_oce_grid_center_lat.long_name='latitude'  ; v_oce_grid_center_lat.bounds="dst_grid_corner_lat"
-v_oce_grid_corner_lon = f_runoff.createVariable ( 'dst_grid_corner_lon', 'f8', ('dst_grid_size', 'dst_grid_corners')  )
-v_oce_grid_corner_lat = f_runoff.createVariable ( 'dst_grid_corner_lat', 'f8', ('dst_grid_size', 'dst_grid_corners')  )
+v_oce_grid_corner_lon = f_runoff.createVariable ( 'dst_grid_corner_lon', 'f8', ('dst_grid_corners', 'dst_grid_size')  )
+v_oce_grid_corner_lat = f_runoff.createVariable ( 'dst_grid_corner_lat', 'f8', ('dst_grid_corners', 'dst_grid_size')  )
 v_oce_grid_corner_lon.units="degrees_east"
 v_oce_grid_corner_lat.units="degrees_north"
 v_oce_grid_area       = f_runoff.createVariable ( 'dst_grid_area'  , 'f8', ('dst_grid_size',) )
@@ -395,25 +395,25 @@
     v_atmOcean        = f_runoff.createVariable ( 'atmOcean'       , 'i4', ('src_grid_size',) )
     v_atmOceanFrac    = f_runoff.createVariable ( 'atmOceanFrac'   , 'i4', ('src_grid_size',) )
     
-    v_atmLand[:]         = atmLand
-    v_atmLandFrac[:]     = atmLandFrac
-    v_atmLandFiltered[:] = atmLandFiltered
-    v_atmFrac[:]         = atmFrac
-    v_atmOcean[:]        = atmOcean
-    v_atmOceanFrac[:]    = atmOceanFrac
+    v_atmLand[:]         = atmLand.ravel()
+    v_atmLandFrac[:]     = atmLandFrac.ravel()
+    v_atmLandFiltered[:] = atmLandFiltered.ravel()
+    v_atmFrac[:]         = atmFrac.ravel()
+    v_atmOcean[:]        = atmOcean.ravel()
+    v_atmOceanFrac[:]    = atmOceanFrac.ravel()
 
 v_atmCoast         = f_runoff.createVariable ( 'atmCoast'       , 'i4', ('src_grid_size',) ) 
-v_atmCoast[:]      = atmCoast
+v_atmCoast[:]      = atmCoast.ravel()
 
 v_oceLand          = f_runoff.createVariable ( 'oceLand'         , 'i4', ('dst_grid_size',) )
 v_oceOcean         = f_runoff.createVariable ( 'oceOcean'        , 'i4', ('dst_grid_size',) )
 v_oceOceanFiltered = f_runoff.createVariable ( 'oceOceanFiltered', 'f4', ('dst_grid_size',) )
 v_oceCoast         = f_runoff.createVariable ( 'oceCoast'        , 'i4', ('dst_grid_size',) )
 
-v_oceLand[:]      = oceLand
-v_oceOcean[:]     = oceOcean
-v_oceOceanFiltered[:]     = oceOceanFiltered
-v_oceCoast[:]     = oceCoast
+v_oceLand[:]      = oceLand.ravel()
+v_oceOcean[:]     = oceOcean.ravel()
+v_oceOceanFiltered[:]     = oceOceanFiltered.ravel()
+v_oceCoast[:]     = oceCoast.ravel()
 
 f_runoff.sync ()
 
