--- CreateWeightsMask.bash.BCKP	2023-01-03 15:51:46.000000000 +0100
+++ CreateWeightsMask.bash	2023-01-03 15:52:19.000000000 +0100
@@ -305,7 +305,6 @@
     MpiRun="/opt/local/bin/mpirun -n 1"
     PyRun="time"
     ;; 
-    ( * ) exit -1 ;;
 esac
 
 set -x ; set -e
@@ -330,7 +329,7 @@
 
 case ${OCE} in # Periodicity type of ORCA grid
     ( ORCA2.4              ) OcePerio=4.2 ;; # 4 ORCA 6 PALEORCA
-    ( ORCA2*               ) OcePerio=4   ;; # 4 ORCA 6 PALEORCA
+    ( ORCA2*               ) OcePerio=6   ;; # 4 ORCA 6 PALEORCA
     ( eORCA1.2             ) OcePerio=6   ;;
     ( eORCA1.4.0           ) OcePerio=6   ;;
     ( eORCA1.4.2           ) OcePerio=6.2 ;;
